docs/changelog.mdx

```diff
@@ -4,6 +4,27 @@ sidebar_label: 'Changelog'
 
 # Changelog
 
+## Release 0.0.29
+
+_October 9th, 2024_
+
+**New Features**
+- Additional game and environment metadata is now captured during gameplay recording (e.g. Unity version, active input devices, etc.)
+- Bot Sequences can now be paused
+- Recordings are now saved directly within projects as a "Latest Recording" that can be copied and edited
+- Segments can now be seen and played directly from the in-game overlay
+- An `-rgsequencepath` argument can be used to start Unity and run a desired Bot Sequence
+- Paths are now displayed for bot sequences within the in-game overlay
+
+
+**Bug Fixes**
+- Pointer events are now properly captured and replayed in recordings
+- Error messages on Macs regarding screenshot data have been removed
+- Control characters that are not JSON-compatible are now filtered out of the replay data
+- The font used in the SDK no longer conflicts with project fonts
+- Scrollbars on the web client have an improved design
+- Scenario runs are now scrollable when many scenario steps are present
```

Comment:
Also a few fixes to scenario builder bugs:
- "Wait for Key" step properly detects key-held events from previous steps as long as we're inside the start and end window
- "Wait for State" step can compare against non-string values

Reply:
Added!